generators/server/index.js

```diff
@@ -459,6 +459,11 @@ module.exports = class JHipsterServerGenerator extends BaseBlueprintGenerator {
               'docker:db:await': `echo "Waiting for Couchbase to start" && wait-on -t ${WAIT_TIMEOUT} http-get://localhost:8091/ui/index.html && sleep 30 && echo "Couchbase started"`,
             });
           }
+          if (prodDatabaseType === MYSQL) {
```

Comment:
I guess that now we should also add this line if devDatabaseType is Mysql too.
Maybe a liveliness probe in the docker-compose file and in the TestContainer implementation would be more efficient than an additional command

Reply:
Testcontainer doesn’t have this problem.
Docker compose uses sleep at microservices.

A liveliness in entrypoint is nice but can be trick.

Reply:
I remember that I made some liveliness probes for some jh services in one of my previous project, let me check